docs/features/W3cCredentials.md

```diff
@@ -0,0 +1,633 @@
+
+To issue a W3C credential, follow these steps:
+
+1. **Prepare the Credential Data:**
+Ensure the credential data conforms to the VC-DI context.
+
+<details>
+<summary>JSON example</summary>
+
+```jsonc
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/security/data-integrity/v2",
+    {
+      "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#"
+    }
+  ],
+  "type": ["VerifiableCredential"],
+  "issuer": "did:key:z6MkqG......",
+  "issuanceDate": "2023-01-01T00:00:00Z",
+  "credentialSubject": {
+    "id": "did:key:z6Mkh......",
+    "name": "John Doe"
+  },
+  "proof": {
+    "type": "Ed25519Signature2020",
+    "created": "2023-01-01T00:00:00Z",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:key:z6MkqG......#z6MkqG......",
+    "jws": "eyJhbGciOiJFZERTQSJ9..."
+  }
+}
+```
+</details>
+
+2. **Select Credential type**
+The ability to choose the credential type (indy, vc_di) to be issued. The credential type is used to determine the schema for the credential data.
+
+The format to change credential can be seen in the ["Demo Instruction"](https://github.com/sarthakvijayvergiya/aries-cloudagent-python/blob/whatscookin/feat/vc-di-proof/docs/demo/README.md)
+
+3. **Send the Credential:**
+Use the `/issue-credential-2.0/send` endpoint to issue the credential.
+
+<details>
+<summary>JSON example</summary>
+
+```jsonc
+{
+  "auto_issue": true,
+  "auto_remove": false,
+  "comment": "Issuing a test credential",
+  "credential_preview": {
+    "@type": "https://didcomm.org/issue-credential/2.0/credential-preview",
+    "attributes": [
+      {"name": "name", "value": "John Doe"}
+    ]
+  },
+  "trace": false
+}
+```
+</details>
+
+4. **Verify the Response:**
+The response should confirm the credential issuance.
+
+<details>
+<summary>JSON example</summary>
+
+```jsonc
+{
+  "state": "credential_issued",
+  "credential_id": "12345",
+  "thread_id": "abcde",
+  "role": "issuer"
+}
+```
+</details>
+
+### Verify a Credential
+
+To verify a credential, follow these steps:
+
+1. **Prepare the Verification Request:**
+Ensure the request conforms to the verification context.
+
+<details>
+<summary>JSON example</summary>
+
+```jsonc
+{
+  "verifiableCredential": [
+    {
+      "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/data-integrity/v2"
+      ],
+      "type": ["VerifiableCredential"],
+      "issuer": "did:key:z6MkqG......",
+      "issuanceDate": "2023-01-01T00:00:00Z",
+      "credentialSubject": {
+        "id": "did:key:z6Mkh......",
+        "name": "John Doe"
+      },
+      "proof": {
+        "type": "Ed25519Signature2020",
+        "created": "2023-01-01T00:00:00Z",
+        "proofPurpose": "assertionMethod",
+        "verificationMethod": "did:key:z6MkqG......#z6MkqG......",
+        "jws": "eyJhbGciOiJFZERTQSJ9..."
```

Comment:
Same as above, `jws` is an invalid signature format here, it needs to be `proofValue` with a multibase representation of the signature.